src/content/ecosystem-explorer/projects/uc-berkeley-underground-physics-group.md

```diff
@@ -16,7 +16,7 @@ tech:
 image:
   src: /assets/images/65974d64d556a0c37cb3ddbb_logo-ucberkeley-white.png
   alt: "UC Berkeley Underground Physics Group Logo"
-website: "https://underground.physics.berkeley.edu/"
+website:
```

Comment:
@matthendrian - Is this project still alive? The site seems to be dead. If not - What should we do? Remove it?

Reply:
Yes, I believe this project is still alive - I'll look into why this link for the site is broken and see if we can get that fixed on their end. In the meantime, maybe we could link to the case study? 

https://sealstorage.io/uc-berkeley-uses-web3-decentralized-storage-for-neutrino-research-data/